### PR TITLE
add mad wrapper

### PR DESCRIPTION
**Pull request by OriolAbril**
Add a wrapper for the median_abs_deviation function in scipy.stats.

### CODE REVIEW

**Comment by canyon289** [comment]
Thanks :)

**Comment by OriolAbril** [comment]
I will "need" this next week to test that https://github.com/arviz-devs/arviz/pull/1974 correctly computes the mcse for the mad correctly. If there are summary stats or functions that are missing open an issue or send a PR. 

`logsumexp` from scipy.special should probably be added for example. Note that most special functions don't need a wrapper though, they work on dataarrays straight away because they are pointwise: softmax, expit, logit... already work on xarray objects without any wrapper needed.
